portalnet/src/overlay_service.rs

```diff
@@ -999,6 +1000,8 @@ where
         }
     }
 
+    fn process_event(&mut self, _event: EventEnvelope) {}
```

Comment:
Sure!

Reply:
This merely provides a scaffold for the communication mechanism.

The initial thought was that it would enable an overlay to communicate discovered events such as a change in a peer's connection status or enr, and prevent overlays that share the same peer from having to initiate the PING-PONG cycle required to find out the same  information.

Another usecase was raised by @ogenev:
> What we know for now is that we will need data from the beacon network to verify post-merge headers in the history network.

Each overlay would need to have its own way of handling events that it receives, which might mean an additional generic on `OverlayProtocol` like `TEventHandler: EventHandler` where `EventHandler` is a newly defined trait. However a full enumeration of all relevant events and how they should be handled isn't known atm.